vipps-ecom-api.md

```diff
@@ -122,16 +108,28 @@ Document version 2.5.75.
 * [Recommendations regarding handling redirects](#recommendations-regarding-handling-redirects)
 * [Questions?](#questions)
 
-## Flow diagram
+
+## Payment flows
+
+There are many ways to use the Vipps eCom API. For example:
```

Comment:
Here, I'm trying to give them a quick overview of the main ways to use this API. I changed the wording to make it more like "here's what needs to happen".
I'm not 100% about line 119 - vipps checkout.